pkgs/by-name/nv/nvi/debian-patches.nix

```diff
@@ -0,0 +1,170 @@
+# Generated by debian-patches.sh from debian-patches.txt
```

Comment:
where is this script?

Reply:
It's the one in the nixpkgs tree here: https://github.com/NixOS/nixpkgs/blob/master/maintainers/scripts/debian-patches.sh